CONTRIBUTING.md

```diff
@@ -1,84 +1,58 @@
-```
-git push origin <your username>/<your feature name>
-# To make sure "origin" points to CMSgov/beneficiary-fhir-data run
-# git remote -v
-# if a different remote points to CMSgov/beneficiary-fhir-data
-# replace "origin" with that remotes name in the origional command
-```
-In order to obtain permission to do this contact a github administrator or reach out on [#bfd](https://cmsgov.slack.com/archives/C010WDXAZFZ).
-Once pushed, [open a pull request](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request) and use this [PR template](https://github.com/CMSgov/cms-oeda-dasg) found in .github
-
-Please fill out each section of the body of the PR or quickly justify why the section does not apply to this particular change.
-Reviewers will automatically be suggested or requested based on the projects CODEOWNERS file, feel free to add other reviewers if desired.
-Once all automated checks pass and two reviewers have approved your pull request, a code owner from the core team will do a final review and make the decision to merge it in or not.
+### Background
 
-If you have any questions feel free to reach out on the [#bfd](https://cmsgov.slack.com/archives/C010WDXAZFZ) channel in CMS slack!
+BFD exists to enable the CMS Enterprise to drive innovation in data sharing so that beneficiaries and their healthcare partners have the data they need to make informed decisions about their healthcare.
 
-**FAQ**
+We provide a comprehensive, performant, and trustworthy platform to transform the way that the CMS enterprise shares and uses data by providing Coverage, Patient, and EOB FHIR-formatted data.
 
-Q: What if the core team rejects my PR?
+Review the [README](https://github.com/CMSgov/beneficiary-fhir-data/blob/master/README.md) for additional information on the BFD. 
 
-A: The BFD core team will commit to never rejecting a PR without providing a path forward. The developer who put up the PR should review any feedback, and discuss with their product manager the scope of the work that is now outstanding.
 
 ### Contributing changes
 
 Contributions to the BFD are welcome from any party inside CMS.
 Small changes like "good first issues" can be submitted for consideration without any additional process.
 
-Any [substantive change must go though an RFC process](#proposing-substantive-changes) before work on the change itself can start.
+Any [substantive change must go though an RFC process](#proposing-substantive-changes) before work on the change itself can start. 
 
-**FAQ**
-
-Q: How do I know what “first issues” are up for grabs? 
+Any code changes should be properly commented and accompanied by appropriate unit and integration test as per DASG engineering guidelines.
 
-A: First issues will be tracked on the BFD’s jira board and labeled (via versions) “Good First Issue”
+![Alt text](docs/assets/BFDContributionDiagram.png "Diagram")
```

Comment:
@egkaufer Can you export this diagram as an SVG file, instead? That way, it'll be much easier for browsers and screen readers (for vision-impaired folks) to see.